src/sections/Projects/GetNighthawk/index.js

```diff
@@ -52,19 +53,19 @@ const Projects = () => {
         <div className="stats">
           <Row>
             <Col lg={3} md={3} sm={6}>
-              <h3>12</h3>
+              <h3><Counter duration={6} end={12} /></h3>
```

Comment:
Duration for this can be changed to 3. Considering it is a very small number.

Reply:
I agree with this, the duration on these metrics feels a bit longer than it should.